deps/rabbit/src/rabbit_classic_queue_index_v2.erl

```diff
@@ -1292,3 +1291,25 @@ highest_continuous_seq_id([SeqId1, SeqId2|Tail], EndSeqId)
     highest_continuous_seq_id([SeqId2|Tail], EndSeqId);
 highest_continuous_seq_id([SeqId|Tail], _) ->
     {SeqId, Tail}.
+
+write_file_and_ensure_dir(Name, IOData) ->
+    case write_file(Name, IOData) of
+        ok -> ok;
+        {error, enoent} ->
+            filelib:ensure_dir(Name),
+            write_file(Name, IOData)
+    end.
+
+write_file(Name, IOData) ->
+    case file:open(Name, [binary, write, raw]) of
```

Comment:
Not what I meant, you can do `ok = file:write_file(QueueNameFile, <<"VHOST: ", VHost/binary, "\n",...>>, [raw])` directly.